.travis.yml

```diff
@@ -40,6 +40,22 @@ script:
   - flake8 --select=F --ignore=F401,F405,F403,F811,F821,F841
   - git diff --exit-code
 
+  # New format check: every 'glob' must include 'lsort'
```

Comment:
Great! Can you add a little more detail to this comment about what kind of problems happen?